src/theme.less

```diff
@@ -66,8 +66,10 @@
      Overrides
 -------------------*/
 
-.loadUIOverrides() {
-  @import (optional) "@{themesFolder}/default/@{type}s/@{element}.overrides";
+.loadUIOverrides(@importDefault: true) {
+  & when (@importDefault = true) {
```

Comment:
Couldn't we just ask if theme is default? So we are prepared for any other similar issue?

` & when not  (@theme = 'default') `